### PR TITLE
Disable PHP output buffering as per Nextcloud Docs

### PR DESCRIPTION
**Pull request by fletchto99**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-nextcloud/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:
This change adds a sed command to replace the `output_buffering=4096` line with `output_bufering=0` in the dockerfile.

## Benefits of this PR and context:
This disables output buffering which will lead to memory issues when uploading large files as per the [nextcloud docs](https://docs.nextcloud.com/server/stable/admin_manual/configuration_files/big_file_upload_configuration.html?highlight=big%20files#configuring-php)

## How Has This Been Tested?

I ran the command on my nextcloud instance and validated that the output buffering was disabled and the error message went away on the nextcloud interface.

## Source / References:

Nextcloud docs recommendation: https://docs.nextcloud.com/server/stable/admin_manual/configuration_files/big_file_upload_configuration.html?highlight=big%20files#configuring-php

### CODE REVIEW

**Review by Sblop — APPROVED** [review]
Confirm it works.

**Comment by fletchto99** [comment]
Yep! It got rid of the warning for me. I also confirmed the the phone config file looked correct.

**Comment by fletchto99** [comment]
This isn't stale. What are the next steps to get this merged?

**Comment by fletchto99** [comment]
Still not stale and relevant

**Comment by j0nnymoe** [comment]
I believe we're due to rebase this container to php8 soon so will do this change at the same time.

**Comment by aptalca** [comment]
Please resolve the conflicts and add a changelog entry, then we'll merge
Thanks

**Comment by fletchto99** [comment]
Done @aptalca! This should be good to merge 🍦

**Comment by fletchto99** [comment]
Not stale

**Comment by filewalker1** [comment]
not stale

**Comment by fletchto99** [comment]
@aptalca Is there something more I need to do to get this merged? I see it keeps being marked as stale.

**Comment by aptalca** [comment]
@fletchto99 sorry, it got triaged and then missed. We had a huge backlog (still do)

reviewing now

File: README.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ docker run --rm --privileged multiarch/qemu-user-static:register --reset
 Once registered you can define the dockerfile to use with `-f Dockerfile.aarch64`.
 
 ## Versions
-
+* **30.09.22:** - Disabled `output_buffering` as per [nextcloud docs](https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/big_file_upload_configuration.html?highlight=output_buffering#configuring-php)
```
Comment (aptalca) [inline README.md:259]: This line should be added to `readme-vars.yml` as the readme is generated from that

Comment (fletchto99) [reply README.md]: Done! I think this should be good for final review now.

**Review by aptalca — APPROVED** [review]

**Comment by aptalca** [comment]
because it is a temporary dev/test branch that will be deprecated soon

**Comment by HannesJo0139** [comment]
Oh so you gonna switch to php8 on main branch soon? Thats great!

**Comment by nemchik** [comment]
> Oh so you gonna switch to php8 on main branch soon? Thats great!

You can test `lspipepr/nextcloud:24.0.6-pkg-29ff1d2e-pr-219` and comment on https://github.com/linuxserver/docker-nextcloud/pull/219 if you'd like to help us confirm everything is working correctly with the new changes coming to the master branch (including this, php8, and more; https://info.linuxserver.io/issues/2022-08-20-nginx-base/ )

**Comment by HannesJo0139** [comment]
> > Oh so you gonna switch to php8 on main branch soon? Thats great!
> 
> You can test `lspipepr/nextcloud:24.0.6-pkg-29ff1d2e-pr-219` and comment on #219 if you'd like to help us confirm everything is working correctly with the new changes coming to the master branch (including this, php8, and more; https://info.linuxserver.io/issues/2022-08-20-nginx-base/ )

Works like a charm. Encountered no issues.

**Pull request merged**